quantii/src/lib.rs

```diff
@@ -24,8 +24,12 @@
 
 extern crate novuskinc;
 // Missing intrinsics patch.
+extern crate alloc;
 extern crate compiler_builtins_patch;
```

Comment:
The comment should apply to the line below.  Basically, `extern crate alloc` should be moved one line up.